src/libraries/Microsoft.PowerFx.Core/Functions/TexlFunction.cs

```diff
@@ -235,7 +235,7 @@ protected void ValidateArgumentIsMutable(TexlBinding binding, TexlNode arg, IErr
 
         // This method is used for managing "x-ms-dynamic-values" and "x-ms-dynamic-schema" OpenApi extensions in connectors
         // https://learn.microsoft.com/en-us/connectors/custom-connectors/openapi-extensions
-        public virtual async Task<ConnectorSuggestions> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, IServiceProvider services, CancellationToken cts)
+        public virtual async Task<ConnectorSuggestions> GetConnectorSuggestionsAsync(FormulaValue[] knownParameters, int argPosition, IRuntimeConnectorContext context, CancellationToken cts)
```

Comment:
>IRuntimeConnectorContext [](http://example.com/codeflow?start=134&length=24)

Can we keep as IServiceProvider? 

And will that let us move IRCC  out of core?